vuepress/guide/lazy-loading.md

```diff
@@ -45,18 +45,25 @@ function setI18nLanguage (lang) {
   return lang
 }
 
-export function loadLanguageAsync (lang) {
-  if (i18n.locale !== lang) {
-    if (!loadedLanguages.includes(lang)) {
-      return import(/* webpackChunkName: "lang-[request]" */ `@/lang/${lang}`).then(msgs => {
-        i18n.setLocaleMessage(lang, msgs.default)
-        loadedLanguages.push(lang)
-        return setI18nLanguage(lang)
-      })
-    } 
+export function loadLanguageAsync(lang) {
+  // If the same language
+  if (i18n.locale === lang) {
+    return Promise.resolve(lang)
```

Comment:
Don't we need to invoke `setI18nLanguage(lang)` this case?

Reply:
@kazupon can you take a look is in needed here?

Reply:
I think we need `setI18nLanguage(lang)`.